Modules/permissions.py

```diff
@@ -118,10 +118,10 @@ def real_decorator(func):
         log.debug(f"Wrapping a command with permission {permission}")
 
         @wraps(func)
-        async def guarded(context: Context):
+        async def guarded(context: Context, *args):
```

Comment:
this will cause a merge conflict with #53

Reply:
I've just given that a review. Once it's merged, I'll update my branch, there's no issue.